developer-docs-site/docs/tutorials/run-a-local-testnet.md

```diff
@@ -7,19 +7,23 @@ import BlockQuote from "@site/src/components/BlockQuote";
 
 # Run a Local Testnet
 
-You can run a local testnet to test and develop against an Aptos Blockchain. This network runs independently of the Aptos ecosystem and is for testing and development purposes only.
+You can run a local testnet of the Aptos Blockchain. This network runs independently of the Aptos ecosystem and is primarily for testing and development purposes.
 
 <BlockQuote type="info">
-Your local testnet will not be connected Aptos devnet.
+Note: your local testnet will not be connected to the Aptos devnet. It will run on your local machine and be contained there.
 </BlockQuote>
 
 ## Getting Started
 
-You can run a local testnet in two ways: using the Aptos-core source code or Docker. The Aptos-core source code is useful when testing modifications to the Aptos-core code base or Aptos Framework. Docker is particularly useful when building services on top of the Aptos Blockchain or applications on top of the Aptos Framework as there is no build overhead and the ledger state persists across restarts of the network by default.
+You can run a local testnet in two ways, either, using the Aptos-core source code or using Docker:
+1. The Aptos-core source code is useful for testing modifications to the Aptos-core codebase or the Aptos Framework.
+2. The Docker is particularly useful for building services on top of the Aptos Blockchain or the Aptos Framework, as there is no build overhead and the ledger persists across network restarts.
 
-### Using Aptos-core source code
+We describe each method below.
```

Comment:
I think this is superfluous and can be omitted.

Reply:
I like completeness :D